gravitee-apim-console-webui/src/management/integrations/integration-configuration/integration-configuration.component.ts

```diff
@@ -141,4 +145,50 @@ export class IntegrationConfigurationComponent implements OnInit {
       )
       .subscribe();
   }
+
+  public deleteFederatedApis(): void {
+    this.matDialog
+      .open<GioConfirmDialogComponent, GioConfirmDialogData>(GioConfirmDialogComponent, {
+        width: GIO_DIALOG_WIDTH.SMALL,
+        data: {
+          title: 'Delete APIs',
+          content:
+            'This action deletes unpublished Federated APIs linked to this integration. ' +
+            'Please note that deleted APIs cannot be restored.',
+          confirmButton: 'Delete APIs',
+        },
+        role: 'alertdialog',
+        id: 'deleteAPIsConfirmDialog',
+      })
+      .afterClosed()
+      .pipe(
+        filter((confirm) => confirm),
+        switchMap(() => {
+          this.isLoading = true;
+          return this.integrationsService.deleteFederatedAPIs(this.integration.id);
+        }),
+        tap((deletedApisResponse) => {
+          this.isLoading = false;
+          this.snackBarService.success(
+            'Federated APIs have been deleted.\n' +
+              '  \u2022 Deleted APIs: ' +
+              deletedApisResponse.deleted +
+              '\n' +
+              '  \u2022 Skipped APIs: ' +
+              deletedApisResponse.skipped +
+              '\n' +
+              '  \u2022 Errors: ' +
+              deletedApisResponse.errors,
```

Comment:
Use interpolation can be useful here with something like

```typescript
'Federated APIs have been deleted.\n' +
              `  \u2022 Deleted APIs: ${deletedApisResponse.deleted}\n` +
              `  \u2022 Skipped APIs: ${deletedApisResponse.skipped}\n` +
              `  \u2022 Errors: ${deletedApisResponse.errors}`,
```

Reply:
Sure, good idea :) Done